runner/dl_checkpoints.sh

```diff
@@ -482,10 +482,9 @@ echo "Starting livepeer AI subnet model downloader..."
 echo "Creating 'models' directory in the current working directory..."
 mkdir -p models/checkpoints models/StreamDiffusion--engines models/insightface models/ComfyUI--{models,output}
 
-# Ensure 'huggingface-cli' is installed.
-echo "Checking if 'huggingface-cli' is installed..."
-if ! command -v huggingface-cli >/dev/null 2>&1; then
-  echo "WARN: The huggingface-cli is required to download models. Please install it using 'pip install huggingface_hub[cli,hf_transfer]'."
+echo "Checking if 'hf' Hugging Face CLI is installed..."
+if ! command -v hf >/dev/null 2>&1; then
+  echo "WARN: The Hugging Face CLI is required to download models. Please install it using 'pip install huggingface_hub'."
```

Comment:
i guess this is an error

```suggestion
  echo "ERROR: The Hugging Face CLI is required to download models. Please install it using 'pip install huggingface_hub'."
```

Is the installation instructions also up to date? Might be a different pip package now idk